taxi/views.py

```diff
@@ -34,6 +35,26 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer-list")
+    template_name = "taxi/manufacturer_form.html"
```

Comment:
template_name` is unnecessary here. Please check the checklist before you send your task on the review. In other similar cases too

![image](https://user-images.githubusercontent.com/104218550/203133471-df208ea0-f6ff-4ebb-9c89-026727318b0e.png)

Reply:
Agree, fix. But in 34 line  template_name = "taxi/manufacturer_list.html". Its wtote mentor))